sdk/automation/Azure.ResourceManager.Automation/tests/Helpers/ResourceDataHelpers.cs

```diff
@@ -0,0 +1,223 @@
+        public static DscNodeConfigurationCreateOrUpdateContent GetDscNodeConfigurationData(string dscconfigurationName)
+        {
+            var data = new DscNodeConfigurationCreateOrUpdateContent()
+            {
+                Name = "SampleConfiguration.localhost",
+                Source = new AutomationContentSource()
+                {
+                    Hash = new AutomationContentHash("sha256", "6DE256A57F01BFA29B88696D5E77A383D6E61484C7686E8DB955FA10ACE9FFE5"),
+                    Value = @"instance of MSFT_RoleResource as $MSFT_RoleResource1ref { ResourceID = ""[WindowsFeature]IIS""; Ensure = ""Present""; SourceInfo = ""::3::32::WindowsFeature""; Name = ""Web-Server""; ModuleName = ""PsDesiredStateConfiguration""; ModuleVersion = ""1.0""; ConfigurationName = ""SampleConfiguration""; }; instance of OMI_ConfigurationDocument { Version=""2.0.0""; MinimumCompatibleVersion = ""1.0.0""; CompatibleVersionAdditionalProperties= {""Omi_BaseResource:ConfigurationName""}; Author=""vameru""; GenerationDate=""03/30/2017 13:40:25""; GenerationHost=""VAMERU-BACKEND""; Name=""SampleConfiguration""; };",
+                    SourceType = AutomationContentSourceType.EmbeddedContent,
+                    Version = "1.0"
+                },
+                ConfigurationName = "SampleConfiguration",
+                IsIncrementNodeConfigurationBuildRequired = false
+            };
+            return data;
+        }
+        #endregion
+
+        #region Runbook
+
+        public static AutomationRunbookCreateOrUpdateContent GetRunbookData()
+        {
+            var data = new AutomationRunbookCreateOrUpdateContent(AutomationRunbookType.Script)
+            {
+                Location = AzureLocation.EastUS
+            };
+            return data;
+        }
+
+        public static void AssertRunbook(AutomationRunbookData data1, AutomationRunbookData data2)
+        {
+            AssertResource(data1, data2);
+            Assert.AreEqual(data1.Location, data2.Location);
+            Assert.AreEqual(data1.State, data2.State);
+            Assert.AreEqual(data1.Description, data2.Description);
+            Assert.AreEqual(data1.JobCount, data2.JobCount);
+        }
+        #endregion
+
+        #region SourceControl
+        public static AutomationSourceControlCreateOrUpdateContent GetSourceControlData()
+        {
+            var data = new AutomationSourceControlCreateOrUpdateContent()
+            {
+                RepoUri = new Uri("https://dev.azure.com/vinkumar0563/_git/VinKumar-AzureAutomation"),
+                Branch = "sdktest",
+                FolderPath = "/Runbooks/PowershellScripts",
+                IsAutoSyncEnabled = false,
+                IsAutoPublishRunbookEnabled = true,
+                SourceType = SourceControlSourceType.VsoGit,
+                SecurityToken = new SourceControlSecurityTokenProperties()
+                {
+                    AccessToken = "p26mwl3frnfa4l5i6a7zjfog7k75qeac7otyfa76q3ceajmrnjoq",
```

Comment:
this is a key leak, please sanitize this

Reply:
fixed